dockstring_example.ipynb

```diff
@@ -0,0 +1,314 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/syedzayyan/dockstring/blob/main/dockstring_example.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "!apt-get install python3.7"
```

Comment:
I was trying out different things and this worked, so....no? because Python3.7 is a bit outdated.....I think and I could not see anything below Python3.8